src/utils-lgpl/fcplay.c

```diff
@@ -80,7 +80,7 @@ static void usage(void)
 
 void play_samples(char **files, unsigned int card, unsigned int device,
 		unsigned long buffer_size, unsigned int frag,
-		unsigned long codec_id, unsigned int file_count);
+		unsigned long codec_id, unsigned int file_count, unsigned int gapless);
```

Comment:
this should be introduced in gapless patch support not here, this patch should just reformat the code to support future gapless, dont mix two things pls

Reply:
ok, will update the patch.

Thanks,
mohs.